Dockerfile

```diff
@@ -1,6 +1,6 @@
-ARG NODE_VERSION=16.20-bullseye-slim
+ARG NODE_VERSION=22.13.1-bullseye-slim
```

Comment:
there is already `22.14.0-bullseye-slim`, we should use it.

or should we switch to the newer `22.14.0-bookworm-slim` ?

Reply:
22.14.0-bookworm-slim has less CVEs might as well go with that.